cluster/manifests/03-ebs-csi/controller-aws-iam-role.yaml

```diff
@@ -0,0 +1,9 @@
+{{ if eq .ConfigItems.kube_aws_iam_controller_kube_system_enable "true"}}
```

Comment:
We haven't had any issues with it, so I'd drop the config item. We can do this in a separate PR, but I wouldn't bother with it for new things.